Bugsnag/KSCrash/Source/KSCrash/Recording/BSG_KSCrash.h

```diff
@@ -117,7 +117,10 @@
  * @param depth the number of frames to discard from the main thread's stacktrace
  * @return an array of BugsnagThread
  */
-- (NSArray<BugsnagThread *> *)captureThreads:(NSException *)exc depth:(int)depth;
+- (NSArray<BugsnagThread *> *)captureThreads:(NSException *)exc
+                                       depth:(int)depth
+                                   unhandled:(BOOL)unhandled
+                                 sendThreads:(BSGThreadSendPolicy)sendThreads;
```

Comment:
If we make the callers in BugsnagClient compute recordAllThreads and make this the only extra parameter to captureThreads, it avoids KSCrash needing to know anything about BSGThreadSendPolicy values and simplifies the logic as the `notifyInternal` route will always be `sendThreads != never` and the calculation in `collectThreads` is easier to understand in the context of RN.